### PR TITLE
Update copy on job listing pages

### PR DESCRIPTION
**Pull request by swillis**
Add a small section to each job listing that tells candidates how to get in touch with us if they are unsure which role they should apply for.

image

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

**Comment by swillis** [comment]
@jamesshedden quick 👓 and review plz? ❤️

File: app/pages/about/jobs/jobs-page.js [hunk 1]
```diff
@@ -68,9 +68,23 @@ export default class JobsPage extends React.Component {
...
+          <div className='site-container u-padding-Vxl'>
+            <div className='u-size-3of5 u-center'>
+              <h2 className='u-text-heading u-color-dark-gray u-text-light u-text-l u-margin-Bm'>
+                Not sure which role is right for you?
+              </h2>
+              <p className='u-text-xs u-color-dark-gray'>
+                While we work with a small number of recruitment agencies, we
+                really love direct applications. If you're not sure whether a
+                role is a good fit for you, please email us
+                on <a href='mailto:jobs@gocardless.com'>jobs@gocardless.com</a> to find out more!
```
Comment (jamesshedden) [inline app/pages/about/jobs/jobs-page.js]: `email us at` rather than `email us on`? V minor but latter sounds a bit weird?

**Comment by jamesshedden** [comment]
@swillis Nice work bud 👍

**Pull request merged**